<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmReservas</class>
 <widget class="QDialog" name="frmReservas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulário de Reservas</string>
  </property>
  <widget class="QDateEdit" name="txtData">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNome">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>130</number>
   </property>
   <property name="placeholderText">
    <string>Nome do cliente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCEP">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="inputMask">
    <string>99999999</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>CEP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCidade">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>80</number>
   </property>
   <property name="placeholderText">
    <string>Cidade</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmail">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Endereço eletrônico</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtQtde">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Qtde</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAcrescentar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Acrescentar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbVariedade">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>440</width>
     <height>25</height>
    </rect>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>74</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CEP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>164</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>104</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblItens">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>611</width>
     <height>201</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>118</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Merc</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Clone</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descrição</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Qtde</string>
    </property>
    <property name="textAlignment">
     <set>AlignTrailing|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Preço</string>
    </property>
    <property name="textAlignment">
     <set>AlignTrailing|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btIncluir">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>540</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Incluir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUF">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>80</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFone">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>80</number>
   </property>
   <property name="placeholderText">
    <string>Fone / WhatsApp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>134</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fone:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="txtPrazo">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>159</y>
     <width>150</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>164</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prazo Previsto:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtObs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>611</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>437</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Observações:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtData</tabstop>
  <tabstop>txtNome</tabstop>
  <tabstop>txtCEP</tabstop>
  <tabstop>txtCidade</tabstop>
  <tabstop>txtUF</tabstop>
  <tabstop>txtFone</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>txtPrazo</tabstop>
  <tabstop>txtQtde</tabstop>
  <tabstop>cmbVariedade</tabstop>
  <tabstop>btAcrescentar</tabstop>
  <tabstop>tblItens</tabstop>
  <tabstop>btIncluir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
